resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/submit/ClientSuite.scala

```diff
@@ -33,7 +33,7 @@ import org.scalatestplus.mockito.MockitoSugar._
 
 import org.apache.spark.{SparkConf, SparkFunSuite}
 import org.apache.spark.deploy.k8s.{Config, _}
-import org.apache.spark.deploy.k8s.Config.WAIT_FOR_APP_COMPLETION
+import org.apache.spark.deploy.k8s.Config._
```

Comment:
Is it for `SPARK_CONF_DIR_CONFIG_MAP_NAME`? If then, shall we enumerate all instead of wild-card?